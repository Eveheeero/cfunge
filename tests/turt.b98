"TRUT"4(                             v
29*N 0H 1P                          v>
52*1-F 0P 2F 1P 53*1-F             v>
2f+8*8*8*8*8*6*5*1+C 9a*H 5F 9a*L v>
0C 5F                            v>
0P 5B 9a*R aB 1P                v>
I @                             >

This program should draw a pattern like below.
The * is a dot.
The horizontal (black) line should be 25 units long.
	There should be a gap 2/3rd along the line of 2 units.
Then a red line (5 units long) at a straight angle.
	The specs doesn't say if 90 degrees is upwards or downwards. So either is
	allowed.
	The red should be (#ff0001) (a slight slight green tint).
Then another (black) line of length 5 at straight angle.
	This should be 90 degrees *right* from the red line.
The dot (black) should be 5 units above the vertical line.
	Or below, depending on where 90 degrees is.

                         *



---------  --------------.
                         |
                         |
                         |
                         |
                         .-----
