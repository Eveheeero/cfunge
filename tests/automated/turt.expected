UNDEF: Pen is 0 
UNDEF: Position is x=0 y=0 
UNDEF: Heading is 0 degrees
UNDEF: Bounds are (163839998 163839998 ) (-163839998 -163839998 )
DONE, now check image
